### ISSUE TITLE
xplor/cns density maps

### ISSUE THREAD

**Issue by schallerdavid** [arose/nglview#761]
Hello,

I would like to use nglview in jupyter notebook to view density maps in protein binding pockets in xplor/cns formats. It appears that nglview has a problem with displaying my maps
([data.zip](https://github.com/arose/nglview/files/2619386/data.zip)). Other softwares (e.g. MOE, PyMol, Maestro) display the maps correctly.

PyMol
![pymol](https://user-images.githubusercontent.com/37929174/49076324-0114d400-f239-11e8-87cd-c0b03e4e7746.png)

nglview
![nglview](https://user-images.githubusercontent.com/37929174/49076060-7338e900-f238-11e8-9f59-798db81df040.png)

Here is a script I used to display  the maps in jupyter notebook. (nglview 1.1.7)

```bash
import nglview as nv
view = nv.NGLWidget()
view.add_component("/home/david/Documents/protein.pdb")
view.clear()
view.add_cartoon()
view.add_component("/home/david/Documents/map.xplor")
view.component_1.update_surface(isolevel=30.0, isolevelType='value', color='green')
view
```

x-plor and cns maps are saved with xy-planes section for section. In my map is a single entry at the last line of each xy-section (see below). May the problem arise because the number of xy-plane grid points is not dividable by 6 without a remainder? Here is the transition from the first to the second xy-section in my maps:
...
 7.98403E-02 6.98603E-02 8.98204E-02 4.99002E-02 2.99401E-02 2.99401E-02
 1.99601E-02 3.99202E-02 4.99002E-02 1.09780E-01 1.49701E-01 1.49701E-01
 9.98004E-02
       1
 7.18563E-01 7.58483E-01 5.08982E-01 2.29541E-01 2.99401E-02 9.98004E-03
 1.99601E-02 5.98802E-02 9.98004E-02 1.29741E-01 2.79441E-01 3.99202E-01
 4.79042E-01 5.58882E-01 5.68862E-01 4.79042E-01 3.29341E-01 2.39521E-01
...

Thanks in advance,

David

**Comment by hainm** [comment]
@arose  Do you have any comment here? I am not an expert in this.

**Comment by hainm** [comment]
@schallerdavid I can not reproduce the nglview's figure you posted. Below is mine.

<img width="1197" alt="screen shot 2018-11-27 at 10 47 23 pm" src="https://user-images.githubusercontent.com/4451957/49127717-782c8580-f296-11e8-8f77-d2436b798880.png">

**Comment by hainm** [comment]
PS: You can use wireframe=True to make the display nicer.

<img width="1186" alt="screen shot 2018-11-27 at 10 49 15 pm" src="https://user-images.githubusercontent.com/4451957/49127785-b9249a00-f296-11e8-9234-482b0455d70c.png">

**Comment by schallerdavid** [comment]
Sorry, I sent the wrong zip. Here is the one with incorrectly displayed maps ([erroneous_maps.zip](https://github.com/arose/nglview/files/2623243/erroneous_maps.zip)). The one I sent before contains maps where the number of grid points of each xy-section is dividable by 6 without a reminder. Those work also in my case.

**Comment by schallerdavid** [comment]
@hainm I can see the map using nglview, but it is not correctly displayed. It looks like the example pics I provided in my first post comparing the pymol and nglview display. Pymol shows the correct display, nglview shows these "flat coins". Just double checked and downloaded the erroneous_maps zip. Don't know why you can not see a map at all.

**Comment by hainm** [comment]
Should be fixed in #769

**Comment by schallerdavid** [comment]
Thanks!
